### PR TITLE
feat: add email as optional params for user registration

### PR DESCRIPTION
**Pull request by yundifu**
# Summary
- add `email` as optional registration params to match with public api

# Why the changes

# Things worth calling out

### CODE REVIEW

**Review by maksim-tsvetkov — APPROVED** [review]
👍

File: src/workflows/workflows.ts [hunk 1]
```diff
@@ -77,8 +78,8 @@ export class Workflows {
     this.withdrawalsApi = new WithdrawalsApi(config.api);
   }
 
-  public registerOffchain(signer: Signer, starkWallet: StarkWallet) {
-    return registerOffchainWorkflow(signer, starkWallet, this.usersApi);
+  public registerOffchain(signer: Signer, starkWallet: StarkWallet, options?: RegistrationOptions) {
```
Comment (joaosf) [inline src/workflows/workflows.ts:81]: Update readme with this parameter please

**Review by joaosf — CHANGES_REQUESTED** [review]

File: src/workflows/workflows.ts (hunk 1 above)
Comment (yundi-fu) [reply src/workflows/workflows.ts:81]: i am not 100% sure if we want to explicitly put this option in the readme, what do you think @maksim-tsvetkov

**Comment by Kastet** [comment]
@yundifu is this PR still relevant? If not, let's close it?

**Comment by yundifu** [comment]
close it for now

**Pull request closed without merging**